packages/swapper/src/swappers/osmosis/OsmosisSwapper.ts

```diff
@@ -184,7 +184,13 @@ export class OsmosisSwapper implements Swapper<ChainId> {
     return {
       buyAmount,
       buyAsset,
-      feeData: { fee, tradeFee },
+      feeData: {
+        fee,
+        tradeFee,
+        networkFee: fee,
+        sellAssetTradeFeeUsd: '0',
+        buyAssetTradeFeeUsd: tradeFee,
```

Comment:
we could propagate the vernacular through to `getRateInfo` as well. had to look just to verify it was def the `buyAssetTradeFee`. at first glance it also looks like this particular trade fee is in crypto value vs fiat as well...

Reply:
Resolving as this is now using the `buyAssetTradeFeeUsd` vernacular, making it clear this refers to fiat trade fee

Reply:
To clarify on this, I was stating that the actual return value from the osmosis `getInfoFromPool` is a crypto value vs fiat:
```
  const tradeFeeBase = bnOrZero(buyAmount).times(bnOrZero(pool.poolParams.swapFee))
  const tradeFee = tradeFeeBase.dividedBy(bn(10).exponentiatedBy(OSMOSIS_PRECISION)).toString()
  ```